source/site-administrators/studio/user-interface-configuration.rst

```diff
@@ -0,0 +1,778 @@
+:is-up-to-date: True
+
+.. index:: User Interface Configuration
+
+.. _user-interface-configuration:
+
+############################
+User Interface Configuration
+############################
+
+The user interface configuration file defines the widgets shown in the user interface.  It allows the user to configure
```

Comment:
Maybe add that the UI is made up of configurable widgets and can be extended or modified by adding/removing/configuring widgets. Plugins can make use of this by adding themselves to the UI where required.